lib/ephemeral/models/build.rb

```diff
@@ -3,7 +3,11 @@
 module Ephemeral
   module Models
     class Build
-      attr_accessor :id, :image, :repo, :build_type, :status
+      attr_accessor :id, :image, :repo, :build_type, :status, :logs
+      
+      def logs
```

Comment:
This shouldn't be here either. This will cause the tests to fail. This entire method doesn't do anything.